src/main/app/src/Components/Landscape/Modals/Item/Item.tsx

```diff
@@ -1,116 +1,204 @@
-import React, { useEffect, useState, ReactElement } from 'react';
+import React, { useState, ReactElement, useEffect } from 'react';
+import { Card, CardActions, CardHeader, Paper, Typography } from '@material-ui/core';
 import { get } from '../../../../utils/API/APIClient';
-import './Item.scss';
-
-import { IItem, IAssessmentProps } from '../../../../interfaces';
-import {
-  getLabels,
-  getLinks,
-  getRelations,
-  getAssessmentSummary,
-  getItemIcon,
-} from '../../Utils/utils';
+import CardContent from '@material-ui/core/CardContent';
+import { IAssessmentProps, IItem } from '../../../../interfaces';
+import { getItemIcon, getLabels, getLinks } from '../../Utils/utils';
+import Button from '@material-ui/core/Button';
+import StatusChip from '../../../StatusChip/StatusChip';
+import IconButton from '@material-ui/core/IconButton';
+import { ArrowDownward, ArrowUpward, FilterCenterFocus, MoreVertSharp } from '@material-ui/icons';
+import componentStyles from '../../../../Ressources/styling/ComponentStyles';
 
 interface Props {
-  fullyQualifiedItemIdentifier: string;
-  findItem?: (fullyQualifiedItemIdentifier: string) => void;
-  onAssessmentClick?: (fullyQualifiedItemIdentifier: string) => void;
-  item?: IItem;
   small?: boolean;
+  useItem?: IItem;
+  findItem?: Function;
+  fullyQualifiedItemIdentifier?: string;
 }
 
 /**
- * Returns a choosen Landscape Item if informations are available
+ * Returns a chosen Landscape item if information is available
+ *
+ *
  */
-const Item: React.FC<Props> = ({ fullyQualifiedItemIdentifier, findItem, onAssessmentClick }) => {
-  const [item, setItem] = useState<IItem | undefined>();
+const Item: React.FC<Props> = ({ useItem, findItem, fullyQualifiedItemIdentifier, small }) => {
   const [assessment, setAssessment] = useState<IAssessmentProps[] | undefined>(undefined);
+  const [item, setItem] = useState<IItem | undefined>(undefined);
+  const [compact, setCompact] = useState<boolean>(false);
+  const classes = componentStyles();
+  let relations: ReactElement[] = [];
 
   useEffect(() => {
-    get(`/api/${fullyQualifiedItemIdentifier}`).then((item) => {
+    const loadAssessment = (item: IItem) => {
+      const landscapeIdentifier = item ? item.fullyQualifiedIdentifier.split('/') : [];
+
+      if (landscapeIdentifier[0]) {
+        //TODO load direct to with fqi in path
```

Comment:
I also don't understand this `TODO`. I just can't parse the sentence :) What needs to be done here?

Reply:
I meant it is not necessary to split strings since each item has a fullyQualifiedIdentifier that can be used directly as API call path element. Also, the complete data has already been loaded (but not by the search component)

Reply:
So we could just do `get(`/assessment/${landscapeIdentifier}`)` directly and our backend already supports that?

Or what has to be done to be able to do that call?

Regarding the already loaded data: do you mean that this call is not really necesary because another call to the backend already retrieved the data this call would retrieve?